[FR] Update `Community\Project_Loved`, `How_you_can_help!`, `Community_Mentorship_Program`, Add `Community`, `Forum`, `Feature_Requests`, `Help`, `Modding_Queues`, `Off-Topic`, `Video_series` (#6524)
ppy/osu-wiki: wiki/Community/fr.md

# Communauté

Cette page répertorie les moyens par lesquels les personnes de tout le site d'osu! peuvent entrer en contact les unes avec les autres. Si vous êtes à la recherche d'opportunités de contribution, mais ne savez pas par où commencer, assurez-vous de lire [Comment vous pouvez aider !](/wiki/Community/How_you_can_help!).

> ## Places
> 
> - [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)

Author: ```suggestion
## Places

- [Forum](/wiki/Community/Forum)
- [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
```

# Communauté

> Cette page répertorie les moyens par lesquels les personnes de tout le site d'osu! peuvent entrer en contact les unes avec les autres. Si vous êtes à la recherche d'opportunités de contribution, mais ne savez pas par où commencer, assurez-vous de lire [Comment vous pouvez aider !](/wiki/Community/How_you_can_help!).

Reviewer: ```suggestion
Cette page répertorie les moyens par lesquels les utilisateurs de tout le site d'osu! peuvent entrer en contact les uns avec les autres. Si vous êtes à la recherche d'opportunités de contribution, mais ne savez pas par où commencer, assurez-vous de lire [Comment vous pouvez aider !](/wiki/Community/How_you_can_help!).
```

## Places

- [Forum](/wiki/Community/Forum)
- [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
- [Serveur Discord osu!dev](/wiki/Community/osu!dev_Discord_server)
- [Organisations hors ligne](/wiki/Community/Organisations)

## Projets

*Voir également : [Liste de tous les projets](/wiki/Community/Projects)*

- [Community mentorship program](/wiki/Community/Community_Mentorship_Program)
- [Mappers' Guild](/wiki/Community/Mappers_Guild)
- [osu! wiki](/wiki/osu!_wiki)
- [Project Loved](/wiki/Community/Project_Loved)

## Initiatives

- [osu! community meetings](/wiki/Community/osu!_Community_Meetings)

> - [Série de vidéos](/wiki/Community/Video_series)

Reviewer: ```suggestion
- [Séries vidéo](/wiki/Community/Video_series)
```
